pandunia-lekse.csv

```diff
@@ -1928,6 +1965,7 @@ meninge|||eng:meninx, spa:por:meninge, fra:méninge|meninx (meninges)|méninge|m
 meningitis||||meningitis||meningitis||||||||||||||aivokalvontulehdus|zapalenie opon mózgowych
 mente||||advice (counsel)||consejo||||||||||||||neuvo|rada, porada
 menter|||san:मन्त्रिन् (mantrin), hin:मंत्री (mantrī), tha:มนตรี (mon-tri), may:menteri, eng:deu:fra:por:spa:mentor, jpn:メンター (mentā), kor:멘토 (mento), rus:ментор (mentor)|advisor (counselor, mentor)||consejero (asesor)||||||||||||||neuvontantaja (mentori)|doradca (mentor)
+merante|||may:eng:meranti|lauan tree (meranti, balau)|||||||||||||||||
```

Comment:
Need more etymology.

Reply:
I wasn't able to find their common name in any language other than English and Malay. The scientific name is Shorea, but it's named after an official in the British East India Company, which strikes me as eurocentric.

Actually, according to Wiktionary, it looks like "meranti" is also used in Portuguese, so I can add that.

Reply:
OK. I suppose "merante" is some kind of tree that grows in the Malay Archipelago area, so that etymology is then enough.